microsoft/PowerToys
# Forcing build to validate signing and versioning

## Summary of the Pull Request

**What is this about:**
Builds currently could have files slip by that should have been signed but didn't get. This will check all dll's and exe's to be both versioned and signed.

This script has been tested with the failure scenario below to ensure it did fail.  it directly checks files extracted from our built MSI.

**What is included in the PR:** 

**How does someone test / validate:** 
add new dll and not in core.json or comment out one of the signed files in the core.json

You should see a variation of the failure below:
![Screenshot 2022-01-10 085820](https://user-images.githubusercontent.com/1462282/148861264-8a991d94-f070-45f9-9bdd-8868b3bcf170.png)

## Quality Checklist

- [x] **Linked issue:** #11648

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
